.github/workflows/add_prs_to_tech_debt_project.yml

```diff
@@ -0,0 +1,19 @@
+# Calls a reusable workflow in the .github repo,
+# which adds the PR that triggered this to the Technical Debt project board, if it is a depfu one.
+# Passes the 'PSD-AddToProject' GitHub App key and App Id as secrets to the reusable workflow.
+
+name: Add dependencies to technical debt project
+
+on:
+  # Triggered on creation of pull requests with any label,
+  # or when a label is added to an existing pull request.
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  call-workflow-add_to_technical_debt_project:
+    uses: sanger/.github/.github/workflows/add_to_tech_debt_project_reusable.yml@master
+    secrets:
+      app_id: ${{ secrets.ADD_TO_PROJECT_APP_ID_PSD }}
+      app_key: ${{ secrets.ADD_TO_PROJECT_APP_KEY_PSD }}
```

Comment:
Looks like there's no final new line character in this file.  I always try to encourage people to set their IDE up to add the final new line when saving a file.  If you're using VSCode, you can do this by adding the settings:

```
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
```

This will ensure every file always has only exactly one final new line.  If necessary, you can also add exceptions for certain file types.  For example, I've found that some `.erb` files for Ruby views will mess up the rendering of the HTML if a final new line is on the file.  But this is very much not the norm and most files should have a new line on the end, otherwise that final line in the file is not technically a line according to [the POSIX standard definition of a line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206).

Reply:
Thanks Stuart for the detailed reply. 
I have copied the file across from a working repo to all other repos. Strangely, it happened in few places.